src/executables/service/locator/shared.ts

```diff
@@ -0,0 +1,72 @@
+import { execSync } from 'child_process';
+import * as fs from 'fs-extra';
+import * as vscode from 'vscode';
+
+export function getRDetailsFromPath(rPath: string): {version: string, arch: string} {
+    try {
+        const child = execSync(`${rPath} --version`).toString();
```

Comment:
This gives the following error on windows:
`'C:\Program' is not recognized as an internal or external command, operable program or batch file.`

Reply:
Good catch, thank you. I haven't had a chance to develop on windows so I think the windows side is definitely lagging behind the unix side of things